runtime/moonbase/src/lib.rs

```diff
@@ -128,7 +128,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("moonbase"),
 	impl_name: create_runtime_str!("moonbase"),
 	authoring_version: 3,
-	spec_version: 0153,
+	spec_version: 0154,
```

Comment:
I believe the new policy we're trying out is to not bump spec versions in each PR, but to bump them as part of the release process.